Libplanet/Net/Transports/NetMQTransport.cs

```diff
@@ -664,6 +662,8 @@ private void DoReply(object sender, NetMQQueueEventArgs<NetMQMessage> e)
             NetMQMessage msg = e.Queue.Dequeue();
             string identityHex = ByteUtil.Hex(msg[0].Buffer);
 
+            _logger.Debug("Dequeued message. ({identity})", identityHex);
```

Comment:
How about lower this log's level to verbose?

Reply:
```suggestion
            _logger.Verbose("Dequeued message. ({identity})", identityHex);
```